controllers/spiaccesstoken_controller.go

```diff
@@ -151,7 +151,7 @@ func (r *SPIAccessTokenReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, fmt.Errorf("failed to load the token from the cluster: %w", err)
 	}
 
-	lg = lg.WithValues("phase_at_reconcile_start", at.Status.Phase)
+	lg = lg.WithValues("phase_at_reconcile_start", at.Status.Phase, "accesstoken", at)
```

Comment:
this puts the whole object into the log values. The log already has the object key set by the controller manager btw. IMHO it would be better to just add the data that is helping you here instead of dumping whole object into the log messages...